src/HTMLGenerator/converter.py

```diff
@@ -14,40 +15,56 @@ def _get_path_relative_to_username(filename):
     filename_path = Path(filename)
     return str(filename_path.relative_to(s.expanduser()))
 
-def convert(top_level_output_directory, top_level_input_directory, json_with_filepaths, path_to_current_working_directory = Path.cwd()):    
+def write_configuration_file(json_with_filepaths, top_level_output_directory = Path(""), top_level_input_directory = Path(""), path_to_current_working_directory = Path.cwd()):
+    current_configuration_absolute_path = Path(path_to_current_working_directory / CURRENT_CONFIGURATION_PATH)
+    # Writing the top, preload, input and output path to the currentConfiguration.txt file.
+    try:
+        with open(current_configuration_absolute_path, 'w') as current_configuration_file:
+            if "topPath" in json_with_filepaths:
+                topDirectoryPath = json_with_filepaths["topPath"]
+            else:
+                raise Exception("topPath cannot be null/empty")
+            for configuration in json_with_filepaths["configurations"]:
+                current_configuration_file.write(
+                '/top:' + str(top_level_output_directory / topDirectoryPath) + "\n")
+                if "preloadedFiles" in configuration:
```

Comment:
I think `preloadedFiles` can be changed to `preloadFiles` since they are not already preloaded, and we are asking the converter to preload. What do you think? @abahetik

Reply:
Yes that would definetly make more sense.